src/main/kotlin/no/nav/familie/tilbake/vilkårsvurdering/VilkårsvurderingMapper.kt

```diff
@@ -266,7 +266,7 @@ object VilkårsvurderingMapper {
 
     private fun utledIleggRenter(ileggRenter: Boolean?, fagsystem: Fagsystem): Boolean? {
         return when {
-            ileggRenter != null && Fagsystem.BA == fagsystem -> false
+            ileggRenter != null && listOf(Fagsystem.BA, Fagsystem.KONT).contains(fagsystem) -> false
```

Comment:
Hvorfor skal den alltid returnere false/null hvis det gjelder BA/KONT?
(og aldri true)

Reply:
Litt usikker på hvorfor det er gjort sånn siden jeg bare har utvidet eksisterende logikk.

Det er vel sånn at `ileggRenter` hentes fra AktsomhetDto som er uavhengig av fagsystem.
Og man for barnetrygd og kontantstøtte aldri ønsker å enten regne ut renter. 

Om kombinasjonen av aktsomhetDto.ileggRenter == true og & fagsystem == KONT || BA kan oppstå vet jeg ikke, men det kan hende at dette ble lagt inn som et sikkerhetsnett for det.